spec/factories/remove_factory.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :remove do
+    explanatory_note { "The document is out of date" }
```

Comment:
I think we try to use random stuff where possible, just so the test data isn't static - we use SecureRandom a lot around the factories. Do you think it would be OK with that?

Reply:
Yep. The note can be anything.

Reply:
Have updated. Will fix-up.